src/sentry/static/sentry/app/components/smartSearchBar/index.tsx

```diff
@@ -379,24 +475,25 @@ class SmartSearchBar extends React.Component {
           ? (currIndex + 1) % totalItems
           : 0;
 
-      const [nextGroupIndex, nextChildrenIndex] = findSearchItemByIndex(
-        searchItems,
+      const [nextGroupIndex, nextChildrenIndex] = filterSearchGroupsByIndex(
+        searchGroups,
         nextActiveSearchItem
       );
 
       // Make sure search items exist (e.g. both groups could be empty) and
       // attach the 'active' property to the item
-      if (searchItems[nextGroupIndex] && searchItems[nextGroupIndex].children) {
-        searchItems[nextGroupIndex].children[nextChildrenIndex] = {
-          ...searchItems[nextGroupIndex].children[nextChildrenIndex],
+      if (
+        nextGroupIndex !== undefined &&
+        nextChildrenIndex !== undefined &&
+        searchGroups[nextGroupIndex]?.children
+      ) {
+        searchGroups[nextGroupIndex].children[nextChildrenIndex] = {
```

Comment:
Same here, though this is a little better, at least it creates a new object instead of just setting active to true